### DISCUSSION TITLE
Hower Tooltip

### DISCUSSION THREAD

**Discussion by BIaze420** [TomSchimansky/CustomTkinter#2146] [Q&A] [answered]
I coudln't find any options for hower tooltips in the documentation, are there any solutions with ctk to create for example information icons that show you something when howering over it?

**Comment by Akascape** [comment, marked as answer]
@BIaze420 Yes bro, we have hover tooltips here: https://github.com/Akascape/CTkToolTip

![](https://user-images.githubusercontent.com/89206401/229771216-199727ef-2a01-4ab3-bac6-7e0a5234389e.gif)

It is an external package. More widgets you can find here: https://github.com/Akascape/tkinter-toolkit

**Reply by BIaze420** [reply]
Awesome, thank you!
